### PR TITLE
Added a cl tool to start mycroft to listen: mycroft-listen

### PR DESCRIPTION
**Pull request by FruityWelsh**
==== Related Issues ====
Alternative triggers. #1357 - Added feature to activate via command line
This can be triggered in most desktop enviroments via short cuts

====  Tech Notes ====
Copied mycroft-speak-to format, just it doesn't need any cl option
and replaced the bus message with "mycroft.mic.listen" message.

====  Documentation Notes ====
Activates the mycroft listening feature.

 On branch feature/mycroft-listen_cli_tool
 Changes to be committed:
	new file:   mycroft-listen

## Description
(Description of what the PR does, such as fixes # {issue number})

## How to test
(Description of how to validate or test this PR)

## Contributor license agreement signed?
CLA [ ] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)

### CODE REVIEW

File: bin/mycroft-listen [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Mycroft AI Inc.
```
Comment (forslund) [inline bin/mycroft-listen]: Should be 2020

File: bin/mycroft-listen [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+SOURCE="${BASH_SOURCE[0]}"
+cd -P "$( dirname "$SOURCE" )"
+DIR="$( pwd )"
+
+# Enter the Mycroft venv
+source "$DIR/../venv-activate.sh" -q
+
+# Send a message to be spoken
+output=$(python -m mycroft.messagebus.send "mycroft.mic.listen")
```
Comment (forslund) [inline bin/mycroft-listen]: Since the output isn't used maybe just do
`python -m mycroft.messagebus.send "mycroft.mic.listen" > /dev/null`

Comment (FruityWelsh) [reply bin/mycroft-listen]: That makes sense, no reason to store unnecessary data.

File: bin/mycroft-listen (hunk 1 above)
Comment (FruityWelsh) [reply bin/mycroft-listen]: Good catch. I agree to that

**Comment by forslund** [comment]
Hi, will you be pushing an update along the lines we discussed earlier?

**Comment by FruityWelsh** [comment]
> Hi, will you be pushing an update along the lines we discussed earlier?

So I am having some issues actually pushing this change.

I manged to update to the ```dev``` branch, but my actually changes:
```# Copyright 2020 Mycroft AI Inc.``` 
and
```python -m mycroft.messagebus.send "mycroft.mic.listen"```
don't seem to pushing when I edit them, do ```git add .``` and ```git push my-mycroft-repo```.

Which is odd as ```git pull upstream dev``` and ```git push my-mycroft-repo``` did work...

**Comment by ChanceNCounter** [comment]
> I forgot to commit lol ...

Welcome to my life!

**Comment by forslund** [comment]
Thanks @FruityWelsh, merging now.

**Pull request merged**